code/modules/admin/topic.dm

```diff
@@ -2868,9 +2868,12 @@
 				SSblackbox.record_feedback("tally", "admin_secrets_fun_used", 1, "Traitor All ([objective])")
 
 				for(var/mob/living/carbon/human/H in GLOB.player_list)
-					if(H.stat == 2 || !H.client || !H.mind) continue
-					if(is_special_character(H)) continue
-					//traitorize(H, objective, 0)
+					if(H.stat == 2 || !H.client || !H.mind)
+						continue
+					if(is_special_character(H))
+						continue
+					if(jobban_isbanned(H, ROLE_TRAITOR) || jobban_isbanned(H, "Syndicate"))
+						continue
```

Comment:
I am 99% sure this is checking for the same thing twice.

Reply:
Different checks. `ROLE_TRAITOR` means actual traitor itself, `Syndicate` is blanket for job banned from *all* special roles. I hate that its named syndicate, dont worry.